isPrime/isPrime.cpp

```diff
@@ -0,0 +1,39 @@
+//4# Prime Number
+
+#include <iostream>
+using namespace std;
+
+void main()
+{
+	int number = 0;
+	int a = 0;
+	int status = 1;
```

Comment:
Change this to boolean if it only represents two values. Also change its name to something understandable.
```suggestion
	bool isPrime= true;
```